aio/content/navigation.json

```
@@ -1068,19 +1068,23 @@
       "title": "Idiomas",
       "children": [
         {
-          "title": "简体中文版",
+          "title": "Inglés",
+          "url": "https://angular.io/"
+        },
+        {
+          "title": "Chino simplificado",
           "url": "https://angular.cn/"
         },
         {
-          "title": "正體中文版",
+          "title": "Chino tradicional",
           "url": "https://angular.tw/"
         },
         {
-          "title": "日本語版",
+          "title": "Japonesa",
```

Sugeriría que la palabra fuera "japonés" ya que se está refiriendo al idioma.
```suggestion
          "title": "Japonés",
```